bundles/org.eclipse.swt/Eclipse SWT PI/win32/library/build.bat

```diff
@@ -108,9 +99,6 @@ GOTO :EOF
 @rem Find Visual Studio
 @rem %1 = path template with '$MSVC_VERSION$' and '$MSVC_EDITION$' tokens
 :FindVisualStudio
-	@rem Early return if already found
-	IF NOT "%MSVC_HOME%"=="" GOTO :EOF
```

Comment:
Why did you remove this?
I understand that for "auto" it will now do a lot more checking and produce a lot more output, finally selecting *last* found valid VS instead of the first one

Reply:
Less lines of code and thus a simpler script.

> I understand that for "auto" it will now do a lot more checking and produce a lot more output, 

It checks a few directories more, but I bet nobody will ever notice those few milliseconds (if at all) saved.
And no there is no additional print-out as you can see in the docs.

> finally selecting last found valid VS instead of the first one

This was not intended and only happened because I accidentally also removed the last `IF NOT "%MSVC_HOME%"=="" GOTO :EOF` check too. But that's restored.

Reply:
> But that's restored

Not sure what you mean, patch still removes this condition.

Reply:
Yes, but only for the first two functions, but the third one will still ensure that the first match found is used, but with less code.